Orlando Walking Tours/Controllers/AddTourVC.swift

```diff
@@ -123,9 +129,8 @@ class AddTourVC: UIViewController, UITextFieldDelegate
             if let navController = segue.destinationViewController as? UINavigationController,
                let vc = navController.topViewController as? LocationListVC
             {
-                // By the time I get here, that Tour object is nil
-                print("sender as tour == \(sender as? Tour)")
-                vc.tour = sender as? Tour
+                self.tour = Tour.MR_findFirstByAttribute("uuid", withValue: (self.uuid?.UUIDString)!)
```

Comment:
(self.uuid?.UUIDString)! should be replaced with self.uuid?.UUIDString ?? ""

that way its better to fail to retrieve an object than to crash the app.